modules/ztp-filtering-ai-crs-using-siteconfig.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-filtering-ai-crs-using-siteconfig_{context}"]
+= Filtering custom resources using SiteConfig filters
+
+You can configure `SiteConfig` custom resources (CRs) to include or exclude CRs for use in the installation phase of the zero touch provisioning (ZTP) GitOps pipeline.
+
+You define a set of extra manifests for inclusion in the cluster install. These manifests are linked to the `SiteConfig` custom resources (CRs) and are applied to the cluster during installation. Using filters, you can easily customize individual `SiteConfig` CRs based on your requirements.
+
+The following example configures a custom single-node cluster `SiteConfig` CR that excludes `extra-manifest/03-sctp-machine-config-worker.yaml` and instead applies a custom `extra-manifest/custom-sctp-machine-config-worker.yaml` file in the install.
+
+.Prerequisites
+
+. Create a Git repository where you manage your custom site configuration data. The repository must be accessible from the hub cluster and be defined as a source repository for the Argo CD application.
+
+.Procedure
+
+. Create a set of extra manifest CRs that the ZTP pipeline uses to customize the cluster installs.
+
+. Change to your custom `/siteconfig` directory. The following example illustrates a sample `/siteconfig` folder:
++
+[source,text]
+----
+siteconfig
+├── site1-sno-du.yaml
+└── extra-manifest
+    └── 03-sctp-machine-config-worker.yaml
```

Comment:
This should not be added to user siteconfig repo. It is part of the source-crs/extra-manifest dir from the ztp repo included in the ztp site generate image by default. 
https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/source-crs/extra-manifest

Reply:
yes, only the `custom-sctp-machine-config-worker.yaml` is needed

Reply:
Updated. I wasn't sure about `- cluster:` from https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/ExtraManifestsFilter.md example  since it is not in http://registry.kni-qe-0.lab.eng.rdu2.redhat.com:3000/kni-qe/ztp-site-configs/src/worker-2-4.10/siteconfig/worker-2.yaml#L33-L36